src/bin/mosaic-start.ts

```diff
@@ -132,6 +133,15 @@ mosaic
 
         await (new Graph(graphDescription).start());
       }
+      // printing of endpoints on console.
+      const endPointsTable = markdownTable([
+        ['', 'rpc port', 'ws port'],
```

Comment:
instead of printing ports, I would personally want to print URL's. Wouldn't want user to apply head to say ws should be starting with ws:// and pthers with https:// ?
we may face challenges to figure out public ips to be used in these URL's, if yes then we could consider printing URL templates. 
@sarvesh-ost views on this ?

Reply:
Yes, agree. Ports are somewhat confusing. URL is better.